### ISSUE TITLE
Pénibilité d'un changement de frenchb pour description, en 2014

### ISSUE THREAD

**Issue by ghost** [labopp/guideb2rm-sphinx#3]
Je fais ce ticket ici mais il concerne tous les documents avec frenchb.

Depuis la 3.0a (je crois 7 mars 2014), frenchb applique ceci:

> — Les étiquettes des listes description sont positionnées comme celles des listes itemize
> et enumerate avec un retrait paramétrable `\listindentFB` par rapport à la marge gauche.

Or il semble que RDT avec TL2013/Debian n'a pas la version de TL2013 finale, (ou encore à l'époque la mise à jour de frenchb ne se faisait pas automatiquement sur TeXLive à cause de problèmes liés à Babel qui a été pendant plusieurs années sans maintenance), bref toutes les listes description produites sur RTD n'ont pas le retrait gauche.

Mais dès qu'on produite en local, elles l'ont, et c'est très frappant visuellement.

RTD:
![capture d ecran 2016-07-01 a 09 59 49](https://cloud.githubusercontent.com/assets/20161540/16515516/b29d8838-3f72-11e6-975a-5b8b242f0e57.png)

TL2014 ou plus récent:
![capture d ecran 2016-07-01 a 10 00 36](https://cloud.githubusercontent.com/assets/20161540/16515528/bd242bb8-3f72-11e6-9ea9-b4fa791b12c5.png)

Ce problème depuis plusieurs années m'enquiquine énormément. frenchb impose la nouvelle norme mais on ne peut PAS simplement l'annuler car le paramètre `\listindentFB` est utilisé aussi pour les autres listes. La doc de frenchb dit de remettre à zéro à **chaque** environnement description :disappointed:

On peut s'en sortir bien sûr mais que c'est pénible. De plus je n'ai pas l'intention que frenchb produise une interface pour son numéro de version, ici encore il faut hacker.

De toute façon il faut décider:
- veut-on le comportement actuel de RTD (vieux frenchb)
- ou le nouveau comportement (TL2014)

dans le premier cas il faut faire quelque chose pour garantir la stabilité future, dans le second il faut hacker `\descriptionFB` en espérant que c'était bien le nom de la macro dans TL2013 (pas vérifié car tout cela prend beaucoup de temps).

**Comment by ktzanev** [comment]
Je suis d'accord avec toi que ces changements dans `franchb` sont pénibles. Par contre je ne sais pas si avec ou sans retraît changera quelque choses pour les lecteurs de cette doc. 

Le nombre de personnes qui penvent être agacés par ça je pense ne dépasse pas 2 ! :smile:

**Comment by ghost** [comment]
il faut dire qu'en effet je me dis qu'il faut que j'arrête urgemment de passer du temps là-dessus !

**Comment by ktzanev** [comment]
Le boss autorise ton départ en vacances sans accès à internet :stuck_out_tongue:, bon travail agent @burnol-math !

**Comment by ghost** [comment]
j'ai cherché pendant vingt minutes un smiley adéquat et là mon voisin se met à faire de gros coups de marteaux, ma vie est vraiment un enfer :repeat: :loop: :weary:
